src/Mvc/Mvc.Core/src/Routing/ConsumesMatcherPolicy.cs

```diff
@@ -65,7 +65,7 @@ public Task ApplyAsync(HttpContext httpContext, CandidateSet candidates)
                 throw new ArgumentNullException(nameof(candidates));
             }
 
-            // We want to return a 415 iff we eliminated ALL of the currently valid endpoints due to content type
+            // We want to return a 415 if we eliminated ALL of the currently valid endpoints due to content type
```

Comment:
The `iff` here appears to be intentional - https://en.wikipedia.org/wiki/If_and_only_if#Distinction_from_%22if%22_and_%22only_if%22